Jenkinsfile

```diff
@@ -190,6 +190,168 @@ pipeline {
             }
         }
 
+        stage('Build Runtime Docker for CUDA 8') {
+            agent {
+                label "nvidia-docker && (mr-dl11||mr-dl16||mr-dl10)"
+            }
+
+            steps {
+                dumpInfo 'Linux Build Info'
+                // Do checkout
+                retryWithTimeout(100 /* seconds */, 3 /* retries */) {
+                    deleteDir()
+                    checkout([
+                            $class                           : 'GitSCM',
+                            branches                         : scm.branches,
+                            doGenerateSubmoduleConfigurations: false,
+                            extensions                       : scm.extensions + [[$class: 'SubmoduleOption', disableSubmodules: true, recursiveSubmodules: false, reference: '', trackingSubmodules: false, shallow: true]],
+                            submoduleCfg                     : [],
+                            userRemoteConfigs                : scm.userRemoteConfigs])
+                }
+
+                script {
+                    CONTAINER_NAME = "h2o4gpu${SAFE_CHANGE_ID}-${env.BUILD_ID}"
+                    // Get source code
+                    withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', credentialsId: "awsArtifactsUploader"]]) {
+                        sh """
+                                nvidia-docker build  -t opsh2oai/h2o4gpu-cuda8-runtime:latest -f Dockerfile-runtime .
```

Comment:
In theory, you should remove the build container after export...

furthermore, do not call it `latest`, since it could bring nasty bugs...i would use proper version which should be fixed per an execution of build (like `1.1.3+master.413`).